.travis.yml

```diff
@@ -0,0 +1,6 @@
+dist: bionic
+
+services:
+  - docker
+
+script: ./scripts/run-build.sh
```

Comment:
Should this refer to `/scripts/build_and_test.sh` instead of `/scripts/run-build.sh`?

Reply:
whoops! fixed.